wagtail/admin/templates/wagtailadmin/account/settings_panels/avatar.html

```diff
@@ -1,35 +1,16 @@
 {% load i18n wagtailadmin_tags %}
 {% block content %}
-    <div class="avatar-panel">
-        <div class="avatar-panel__image">
-            {% avatar user=request.user size="large" %}
+    <div class="avatar-panel w-flex w-flex-wrap w-items-center w-mt-5">
+        <div class="avatar-panel__image w-mr-9 w-mb-5">
+            {% if form.avatar.value %}
+                <input class="avatar-clear" type="checkbox" name="avatar-avatar-clear" id="avatar-avatar-clear_id">{% trans "Reset to default" %}
```

Comment:
When adding a label next to a checkbox, it's better to use the proper `label` `for` attribute, so that the text itself is clickable.

```suggestion
                <input class="avatar-clear" type="checkbox" name="avatar-avatar-clear" id="avatar-avatar-clear_id">
                <label for="avatar-avatar-clear_id">{% trans "Reset to default" %}</label>
```

see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Reply:
I used a label before. But if a label is used, it breaks to the next line. 
Hmm, maybe i should have made both inline??